### PR TITLE
adding GCE support

### PR DESCRIPTION
**Pull request by adamleff**

### CODE REVIEW

File: Berksfile [hunk 1]
```diff
@@ -2,6 +2,7 @@ source 'https://supermarket.chef.io'
 
 cookbook 'chef-ingredient'
 cookbook 'fancy_execute', git: "https://github.com/irvingpop/fancy_execute.git"
+cookbook 'marketplace_gce', git: 'https://github.com/chef-partners/marketplace_gce.git', branch: 'initial-release'
```
Comment (ryancragun) [inline Berksfile]: should we YOLO off of master instead?

File: attributes/gce.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+default['marketplace_image']['gce']['public']['aio']['image_id'] = 'ami-1e2c0a74'
```
Comment (ryancragun) [inline attributes/gce.rb]: You mix `image_id` and `ami_id` and have ami id's for gce instances.

**Comment by ryancragun** [comment]
:100:

File: attributes/gce.rb (hunk 2 above)
Comment (adamleff) [inline attributes/gce.rb]: LOL I hardcoded the centos7 image in the terraform json which is why I didn't catch this. Thanks for that.

**Pull request merged**